frontend/src/scenes/billing/CreditCTAHero.tsx

```diff
@@ -0,0 +1,73 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { BlushingHog } from 'lib/components/hedgehogs'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import useResizeObserver from 'use-resize-observer'
+
+import { billingLogic } from './billingLogic'
+import { PurchaseCreditsModal } from './PurchaseCreditsModal'
+
+export const CreditCTAHero = (): JSX.Element | null => {
+    const { width, ref: heroRef } = useResizeObserver()
+
+    const { selfServeCreditEligibility, isPurchaseCreditsModalOpen } = useValues(billingLogic)
+    const { showPurchaseCreditsModal } = useActions(billingLogic)
+    const { featureFlags } = useValues(featureFlagLogic)
+
+    if (!selfServeCreditEligibility.eligible || selfServeCreditEligibility.status === 'paid') {
+        return null
+    }
+    if (!featureFlags[FEATURE_FLAGS.PURCHASE_CREDITS]) {
+        return null
+    }
+
+    return (
+        <div className="flex relative justify-between items-center rounded-lg bg-mark mb-6" ref={heroRef}>
+            <div className="p-4">
+                {selfServeCreditEligibility.eligible && selfServeCreditEligibility.status === 'pending' && (
+                    <>
+                        <h1 className="mb-0">Your credits are processing</h1>
+                        <p className="mt-2 mb-0 max-w-xl">
+                            You've initiated the process to purchase credits.{' '}
+                            {selfServeCreditEligibility.collection_method === 'send_invoice'
+                                ? "You'll receive an email with a link to pay the invoice. Please make sure to pay that as soon as possible so we can apply the credits to your account."
+                                : "We'll will charge your card on file and we will notify if there are any issues."}{' '}
+                            The credits should be applied ot your account within 24 hours of completing your payment.{' '}
```

Comment:
I think this could maybe be easier to consume, but it's not necessary.

### Your credit purchase of $X is pending.
**Status:** < LemonTag >Awaiting invoice payment | Processing payment | Applying credits</ LemonTag >
**Days left to pay:** 14

Reply:
I don't have this information being passed through yet, I'll make an iteration on this after we get this out.